neclib/devices/thermometer/l_218.py

```diff
@@ -0,0 +1,25 @@
+import astropy.units as u
+
+import ogameasure
+
+from ... import config
+from .thermometer_base import Thermometer
+
+
+class L218(Thermometer):
+
+    Manufacturer = "Lakeshore"
+    Model = "Model218"
```

Comment:
混乱を避けるため、クラス名と `Model` の値を揃えてほしいです。
あと細かいですがメーカー名も正確に書いておきたいです。

```suggestion
class Model218(Thermometer):

    Manufacturer = "LakeShore"
    Model = "Model218"
```

Reply:
これについてですがこのファイルの名前自体も変えておいたほうがいいでしょうか。
l218.pt -> model218.py

Reply:
確かにファイル名・パラメータ名も揃えたほうがいいと思います！